### ISSUE TITLE
Excluding clef sign on 2nd and subsequent staves

### ISSUE THREAD

**Issue by foxbunny** [paulrosen/abcjs#669]
Is there a way to exclude the clef sign on staves other than the first one?

Something like this:

![image](https://user-images.githubusercontent.com/65413/115584251-ee4c3f80-a2ca-11eb-8081-b0e998735e39.png)

**Comment by foxbunny** [comment]
@paulrosen I would like to take a stab at creating a PR for this.

So far I just played around the code base to get a handle on what's where. Let me first relay what I think would be the way to add this feature:

1. `VoiceElement` could have an boolean `omitStaffExtra` property which could be used in its `addChild()` method to ignore elements with type starting with `staff-extra`.
2. In order to set the `omitStaffExtra` to the correct value, `createABCStaff()` needs to be aware of the line number it is working with. This information needs to trickle down from the `EngraverController` where `createABCLine()` is called. The line index `i` can be passed as the third (or maybe second is nicer?) argument to `createABCLine()`, which then trickles it to `createABCStaff()`
3. Engraver would have an option that would be called something like `initialSignature` or something (not sure about the name, as I am not really a musician) that would be looked up in `createABCStaff()` so that `omitStaffExtra` would be set to `true` when this option is `true` *and* line index `i > 0`.

What do you think?

**Comment by paulrosen** [comment]
You can do this purely through the ABC string:
```
X:1
T:My Shining Hour
L:1/4
M:4/4
K:C
"Ebmaj7"z4 | "C7"z4 | "Fm7"z4 | "Bb7"z4 | 
K: C clef=none
"Ebmaj7"z4 | "C7"z4 | "Fm7"z4| "Dm7b5"z4 "G7b9"y | 
```
For features that can be done through the ABC standard I prefer to do them that way.

But if you are interested in contributing on other things, welcome and thanks! There are lots of unfinished features.

**Comment by foxbunny** [comment]
The `clef=none` solution did not work, because if clef is anything other than treble clef, only the first line renders accordingly, and the rest render as if clef were treble.

Here's a comparison:

With `clef=none`:

<img width="539" alt="2021-04-23 20_19_01-Example Generator _ abcjs — Mozilla Firefox" src="https://user-images.githubusercontent.com/65413/115913768-5177d680-a471-11eb-8101-2060c1e21ced.png">

With the proposed patch:

<img width="538" alt="2021-04-23 20_19_29-Example Generator _ abcjs — Mozilla Firefox" src="https://user-images.githubusercontent.com/65413/115913820-648aa680-a471-11eb-8372-53a015811d6c.png">

**Comment by paulrosen** [comment]
How about an option that is passed to `renderAbc` of `{ clefFirstLineOnly=true }`? Your approach sounds reasonable but instead of targeting "staff-extra" I would target the clef only.

**Comment by foxbunny** [comment]
The option name sounds good. And roger on the 'staff-extra', it was an omission on my part while I was exploring the code.

**Comment by paulrosen** [comment]
Fixed in version 6.0.0-beta.35
